src/auth.ts

```diff
@@ -44,6 +46,7 @@ google.options({
 export const script = google.script({version: 'v1'}) as Script;
 export const logger = google.logging({version: 'v2'}) as Logging;
 export const drive = google.drive({version: 'v3'}) as Drive;
+export const discovery = google.discovery({version: 'v1'}) as Discovery;
```

Comment:
@grant not sure if this should be in the auth file, as you don't need to authenticate to use it. However, it made sense at the time to put it there.